### PR TITLE
Extends Incinerator Maintenance

### PR DESCRIPTION
**Pull request by Norgad**
A complete remake/redo of #4308 based on feedback and more mapping knowledge.

:cl: Norgad
add:  Maintenance around the incinerator has been expanded. 
add: Engineering maintenance finally has an APC.
add: The incinerator has been moved eastward, there is now a construction area in it's previous location.
/:cl:

map

### CODE REVIEW

**Comment by M3hillus** [comment]
(+1) I like it. Huge improvement over the last iteration you attempted.

**Comment by Ssirius** [comment]
More murdermaint is never a bad thing, more room for, "fun" activities.

:+1:

**Comment by Bigtimetripod** [comment]
Finally... i can be more stealth then ever.

**Comment by ProperPants** [comment]
I like that spooky dungeon shower right below cargo. The nightmares that could happen there... :fearful: :shower:

**Comment by monster860** [comment]
I still don't really like having maintenance pass through the mechanic's podbay.

**Comment by pinatacolada** [comment]
those windows in the room with the tables and chairs look redundant, windows towards a wall?

**Comment by Vivalas** [comment]
I would remove the main passing through the podbay but it looks great otherwise.

**Comment by Twinmold93** [comment]
For the construction zone area you have, either both doors should be general access or both should be engineering access, not one and the other. Personally, I'd go with general access, to allow more open flow. Maintenance shouldn't pass through the pod bay, as it seems to make the pod bay look... Bulky, too long. The maint just to the south of pod bay is unnecessary. The window is fine. Other places on the station have done this (https://imgur.com/Jr514yi), and it does give a little bit of balance to that maintenance tunnel as it previously didn't have a window.

**Comment by Norgad** [comment]
@Twinmold93 The door has been changed to maintenance access

**Comment by Spacemanspark** [comment]
I like it.

**Comment by taukausanake** [comment]
I think if you want maint attached to Mechanic Bay then you might as well give them the access to enter it. It's a type of good redundancy as it gives them another way out of their area. That means though that no one else should have that access though.

This is better than the previous PR made and I can see the new maint hall being put to use. The only thing I can think of is how the incinerator is a bit of an eyesore now sticking out like that and giving the pod bay such a small area to use entering and exiting. Incinerator should be moved up a bit and maybe rearranged to be more to itself than out like that

**Comment by Fox-McCloud** [comment]
That huge square room is ugly as hell, I'm sorry generally speaking you want to try to avoid big square empty rooms like that. 

Yes, I realize that's what the assembly line is, but the way it's decorated and designed doesn't make it feel that way. I'd recommend doing something similar to that so it doesn't feel like engineering construction room 2.0 (which is a terrible room).

**Comment by Norgad** [comment]
@taukausanake To make the podbay less restricted for takeoff and landing, the solar array has been moved westward. This allows pods to cut around, and take a much shorter route southward around the station

**Comment by Norgad** [comment]
Mapmerge is having a strop. Ill fix it later

**Comment by Da-Dman234** [comment]
Not that this is a massive concern, but a lot of the maint you added is restricted to single tile hallways without much of a place to go. In general it's better to have a few more places where it's two or three tiles long, but like I said, not a huge problem.

**Comment by Norgad** [comment]
Working om modifying the maint layout a bit.

@taukausanake The mechanic now has a door into maint from the parking area. There should be no risk of pod-thieves hacking the door and taking the pod, as you still require mechanic access to open the pod-doors.

@Fox-McCloud The construction are looks nicer now.

**Comment by Norgad** [comment]
@TheDZD I'm done on this for major changes. As long as nobody brings up any large issues with this within the next day or so, it's ready to merge.

**Comment by Norgad** [comment]
@Fox-McCloud I just fixed it half an hour ago

**Comment by Norgad** [comment]
# OH GOD WHAT HAVE I DONE

If anyone knows how to make git overwrite a file on my branch with the copy from the master, **_please_** tell me how. Knowing how to do that would make fixing merge conflicts a thousand times easier.

**Comment by Taaketa** [comment]
I'm very excited for this to merge. Maint is gonna get a new Vox bar.

**Comment by taukausanake** [comment]
Can we get an updated image of the area before this is finalized? Sounds like you changed enough without updating the OP

**Comment by Norgad** [comment]
@taukausanake The image has been updated every time a commit changes the map.

**Comment by taukausanake** [comment]
@Norgad Didn't look like it. Pod bay still has a small space to work with but I guess that's barely an issue now

**Comment by Norgad** [comment]
@taukausanake Your original comment way up above was very strangely worded. Sorry if I misinterpreted it, but I interpreted it to mean 'give the mechanic a maint door'

**Comment by Spacemanspark** [comment]
It's borked

**Comment by taukausanake** [comment]
It's dead. Necro it with bluespace

**Comment by Norgad** [comment]
Guys please. I don't know how this went so wrong ;v;

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -8771,6 +8771,72 @@
 "dmI" = (/turf/space,/turf/simulated/shuttle/wall{icon_state = "swall_f9"; dir = 2},/area/shuttle/trade/sol)
 "dmJ" = (/obj/docking_port/stationary{dir = 8; dwidth = 2; height = 11; id = "trade_dock"; name = "port bay 4 at Cyberiad"; width = 5},/turf/space,/area/space)
 "dmK" = (/obj/machinery/door/airlock/external{id_tag = "trade_dock"},/turf/simulated/floor/plating,/area/hallway/secondary/entry)
+<<<<<<< HEAD
```
Comment (VampyrBytes) [inline _maps/map_files/cyberiad/cyberiad.dmm]: you've still got merge conflict code in here...

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -8771,6 +8771,72 @@
...
+"dnO" = (/obj/item/weapon/wrench,/turf/simulated/floor/plating,/area/construction/altmaint)
+"dnP" = (/obj/item/weapon/table_parts,/turf/simulated/floor/plasteel,/area/construction/altmaint)
+"dnQ" = (/obj/item/stack/rods{amount = 8},/turf/simulated/floor/plating,/area/construction/altmaint)
+"dnR" = (/turf/simulated/wall/r_wall,/area/construction/Storage)
+"dnS" = (/obj/machinery/door/airlock/maintenance{name = "Mechanic Workshop Maintenance"; req_access = 70},/turf/simulated/floor/plating,/area/maintenance/aft)
+"dnT" = (/obj/structure/closet/firecloset,/turf/simulated/floor/plating,/area/maintenance/apmaint)
+"dnU" = (/obj/structure/closet/crate,/obj/effect/spawner/lootdrop/maintenance,/turf/simulated/floor/plating,/area/maintenance/apmaint)
+"dnV" = (/obj/machinery/light/small,/turf/simulated/floor/plating,/area/maintenance/apmaint)
+"dnW" = (/obj/machinery/atmospherics/unary/outlet_injector/on{dir = 4},/turf/simulated/floor/plating/airless,/area/maintenance/incinerator)
+=======
```
Comment (VampyrBytes) [inline _maps/map_files/cyberiad/cyberiad.dmm]: here

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 3]
```diff
@@ -8796,6 +8862,7 @@
 "dnh" = (/obj/structure/disposalpipe/segment{dir = 8; icon_state = "pipe-c"},/turf/simulated/floor/plasteel,/area/quartermaster/office)
 "dni" = (/obj/machinery/light{dir = 8},/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/structure/disposalpipe/segment{dir = 4},/turf/simulated/floor/plasteel{dir = 8; icon_state = "brown"},/area/quartermaster/office)
 "dnj" = (/obj/structure/cable{d1 = 1; d2 = 4; icon_state = "1-4"; tag = ""},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/structure/disposalpipe/sortjunction{dir = 1; icon_state = "pipe-j1s"; sortType = 13},/turf/simulated/floor/plasteel{icon_state = "white"},/area/medical/research{name = "Research Division"})
+>>>>>>> refs/remotes/ParadiseSS13/master
```
Comment (VampyrBytes) [inline _maps/map_files/cyberiad/cyberiad.dmm]: and here

**Comment by Norgad** [comment]
w h a t is even going on?

**Comment by Norgad** [comment]
@TheDZD @Fox-McCloud 
I finally resolved the merge conflict on this.

**Comment by Fox-McCloud** [comment]
the APC in the one room is literally hanging in mid air.

**Comment by Norgad** [comment]
@Fox-McCloud Whoops, the dir on it is correct, but the pixel offset is wrong. Will fix in a sec

**Comment by Norgad** [comment]
@Fox-McCloud Done. Any other issues?

**Comment by TheDZD** [comment]
You see to have some rogue commits now.

**Comment by Fox-McCloud** [comment]
This is good, as far as I'm concerned---but yeah, those rogue commits are no-go.

**Comment by Norgad** [comment]
@TheDZD @Fox-McCloud How would I fix the rouge commits, and how did they even occur?

**Comment by Citinited** [comment]
I have no experience with mapping, but shouldn't the APC in the old incinerator be mounted on the wall instead of floating in the air?

**Comment by CrAzYPiLoT-SS13** [comment]
It has pixel offsets, Citinited. That means that it's located on the tile
where he put it, but actually, you see it on the wall because of offsets.

**Comment by Norgad** [comment]
Does anyone know how to fix the rouge commits/files?

**Comment by taukausanake** [comment]
You might as well start with a clean branch and start over. I don't think you can get rid of the rouge commits without some amount of starting over. Just find your edits and compare them to a clean version of master. Then move them over, edit the map, and see if that fixes everything

**Comment by Norgad** [comment]
@taukausanake Does that involve making a new PR?

**Comment by taukausanake** [comment]
It would be safer to in my opinion but I'm not a pro Git user yet. I'm just going off that this PR is tainted already and if you want to start clean you start clean. You can get a second opinion and I'd prefer you do but that's just what I would do in this case

**Pull request closed without merging**